cmd/ttn-lw-cli/internal/util/flags.go

```diff
@@ -96,6 +96,19 @@ func FieldMaskFlags(v interface{}, prefix ...string) *pflag.FlagSet {
 	return fieldMaskFlags(prefix, t, true)
 }
 
+func isStopType(t reflect.Type, maskOnly bool) bool {
```

Comment:
The `isAtomicType` was originally intended for use cases like this, why didn't that work for your Struct?

Reply:
`isAtomicType` will not stop the addition of a field if only a mask is requested (see [here](https://github.com/TheThingsNetwork/lorawan-stack/blob/32c4c14ab290a1faaac1b323e2af453e1d0125d9/cmd/ttn-lw-cli/internal/util/flags.go#L185-L195)). If no field mask is provided by the user, the field mask from `util.FieldMaskFlags` will be used, and it will contain the `Fields` field.